tests/xfstests/run.pm

```diff
@@ -331,7 +331,9 @@ sub run {
     save_tmp_file('status.log', $status_log_content);
     my $local_file = "/tmp/opt_logs.tar.gz";
     script_run("tar zcvf $local_file --absolute-names /opt/log/");
-    save_tmp_file('opt_logs.tar.gz', Mojo::File->new($local_file)->slurp());
+    script_run("NUM=0; while [ ! -f $local_file ]; do sleep 10; NUM=\$(( \$NUM + 1 )); if [ \$NUM -gt 5 ]; then break; fi; done");
```

Comment:
I am not familiar with the rest of the code, but I don't get the logic of this command. 
    If file exists: wait (until timeout of 50s)
    if file doesn't exist: do nothing?

 If the file doesn't exists the next command will fail, won't it?